go/tools/builders/nogo_vet.go

```diff
@@ -91,7 +90,7 @@ func splitOutput(out string) []string {
 
 // buildVetcfgFile creates a vet.cfg file and returns its file path. It is the
 // caller's responsibility to remove this file when it is no longer needed.
-func buildVetcfgFile(packageFile, importMap map[string]string, stdImports, files []string) (string, error) {
+func buildVetcfgFile(packageFile, importMap map[string]string, stdImports, files []string) (vcfgPath_ string, err error) {
```

Comment:
Is the underscore in `vcfgPath_` a typo?

Reply:
That's actually intentional. There is a return statement below which sets `vcfgPath_` to `""`. If the deferred closure referenced it, it would not be able to remove the file. Instead, it references `vcfgPath` which is only set in one place. I thought about giving them unique names earlier, but that seemed less readable.